e2e/updatecli.d/warning.d/autodiscovery/cargo/example.1.yaml

```diff
@@ -0,0 +1,23 @@
+name: "Cargo compose autodiscovery using git scm"
+scms:
+  default:
+    kind: git
+    spec:
+      url: https://github.com/loispostula/cargo-lab.git
```

Comment:
I am starting to wonder if we shouldn't create a new Github organization named `updatecli-test` to store those kind of repsitories.

Reply:
That would work well I think

Reply:
I just created https://github.com/orgs/updatecli-test 
That would be great if you could migrate the repository over there as it will be more maintainable